src/Roslyn.Diagnostics.Analyzers/VisualBasic/VisualBasicApplyTraitToClass.vb

```diff
@@ -0,0 +1,29 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Composition
+Imports Analyzer.Utilities
+Imports Microsoft.CodeAnalysis
+Imports Microsoft.CodeAnalysis.CodeRefactorings
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+Imports Roslyn.Diagnostics.Analyzers
+
+Namespace Roslyn.Diagnostics.VisualBasic.Analyzers
+    <ExportCodeRefactoringProvider(LanguageNames.VisualBasic, Name:=NameOf(VisualBasicApplyTraitToClass))>
+    <[Shared]>
+    Public NotInheritable Class VisualBasicApplyTraitToClass
+        Inherits AbstractApplyTraitToClass(Of AttributeSyntax)
+
+        Public Sub New()
+        End Sub
+
+        Private Protected Overrides ReadOnly Property RefactoringHelpers As IRefactoringHelpers
+            Get
+                Return VisualBasicRefactoringHelpers.Instance
+            End Get
+        End Property
```

Comment:
```suggestion
        Private Protected Overrides ReadOnly Property RefactoringHelpers As IRefactoringHelpers = VisualBasicRefactoringHelpers.Instance
```

Reply:
I am not a fan of this form in Visual Basic. The code I wrote is the shortest code that the language supports for the behavior I intended.

Reply:
➡️ Leaving this one

Reply:
I'm with Youssef on this.  The latter form is much nicer and clearer.  Making the singleton type one field larger is well worth it for the savings in verbosity here.  The former form just means so much more boilerplate scaffolding one has to deal with when reviewing the code.  It doesn't earn its weight so to speak.

Reply:
I'm happy to change this to the shorter form if/when https://github.com/dotnet/vblang/issues/61 is implemented (or the code in this project is ported to C#).